### DISCUSSION TITLE
Update whole column at once

### DISCUSSION THREAD

**Discussion by jungle-boogie** [saulpw/visidata#1359] [Q&A] [answered]
Hello,

I have an MS Excel sheet that contains an incrementing number in column A. 

I want to replace all records in column A with a set integer. How can I do this? Most of the editing commands I've found specify how to edit rows, but I want to 'copy cells' in Excel lingo.

Current:
0|foo
1|bar
2|baz
3|boo
4|sss

what I want:
1|foo
1|bar
1|baz
1|boo
1|sss

What do I need to do?

**Comment by jungle-boogie** [comment, marked as answer]
Ah, I figured it out....at least one way.

```
gs
ge 1
```

**Reply by saulpw** [reply]
Yup you got it!
